index.html

```diff
@@ -40,53 +45,50 @@ <h3>LET'S CONNECT</h3>
                 </div>
             </footer>
         </section>
+      </section>
       <section id="my_work">
         <section id="p1">
```

Comment:
- [ ] Kindly add a background color for the body `#E5E5E5` and for the sections set a #FFF background. This is what makes look your project different from the original design.
![Captura de pantalla_2021-09-24_21-11-08](https://user-images.githubusercontent.com/53111464/134754433-9656d04f-c1ce-4a43-8388-fd96e0dc8606.png)

**Figma**
![Captura de pantalla_2021-09-24_21-05-09](https://user-images.githubusercontent.com/53111464/134754413-afa06dd0-dc57-4ddf-8a21-84ef44565355.png)